src/js/controller/instream-html5.js

```diff
@@ -103,7 +103,6 @@ define([
 
                 provider.resetEventListeners();
 
-                provider.addGlobalListener(_forward);
```

Comment:
I would expect this to be replaced by something else, not just removed. Will this work?